auth_from_http_remote_user/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Author: Laurent Mignon
+# Copyright 2014-2018 'ACSONE SA/NV'
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    'name': 'Authenticate via HTTP Remote User',
+    'version': '11.0.1.0.0',
+    'category': 'Tools',
```

Comment:
IMO it's worth a stable [development status key](https://odoo-community.org/page/development-status).
```suggestion
    'category': 'Tools',
    'development_status': "Production/Stable",
```